x-pack/plugins/security_solution/public/common/components/sourcerer/index.tsx

```diff
@@ -205,6 +205,7 @@ export const Sourcerer = React.memo<SourcererComponentProps>(({ scope: scopeId }
             <EuiFlexItem grow={false}>
               <EuiButton
                 onClick={handleSaveIndices}
+                disabled={selectedOptions.length === 0}
```

Comment:
can you please put this value into a `useMemo` so that we do not have unintentional re-renders?

Reply:
done :)